app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,38 @@
+class CommentsController < ApplicationController
+  def index; end
+
+  def new
+    comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: } }
+    end
+  end
+
+  def show
+    @post = Comment.find(params[:id])
+  end
+
+  def create
+    # Find the user who is creating the post
+    @user = current_user
+    @post = Post.find(params[:post_id])
+    # Build a new post associated with the user
+    comment = params[:comment]
+    @new_comment = Comment.new(comment.permit(:text))
+    @new_comment.author = @user
```

Comment:
**OPTIONAL** - You have done well so far, but according to the project requirements, you are to use secure params to prevent your database from being vulnerable to attack. Please configure the strong params as you did in the post model. Please fix this 👍 👍